lib/models/genre.dart

```diff
@@ -0,0 +1,18 @@
+class Genre {
+  final int id;
+  final String genre;
+
+  Genre({
+    required this.id,
+    required this.genre,
+  });
+
+  factory Genre.fromJson(Map<String, dynamic> json) {
+    final id = json['id'];
+    final genre = json['name'];
```

Comment:
Is not necessary to previously store the values on variables, you can send them directly to the constructor

Reply:
done